charts/gatekeeper/templates/gatekeeper-validating-webhook-configuration-validatingwebhookconfiguration.yaml

```diff
@@ -35,6 +35,9 @@ webhooks:
       - {{ . }}
       {{- end }}
     {{- end }}
+    {{- if .Values.validatingWebhookCustomNamespaceSelectors }}
```

Comment:
@jordiprats looks like `charts` is still being modified. can you revert any changes to `charts` please?